packages/bitcore-node/migrations/fixUnspentInputs.js

```diff
@@ -0,0 +1,65 @@
+/****************************************
```

Comment:
Add a shebang to the top of the file `#!/usr/bin/env node` and give this file execution permissions `chmod +x fixUnspentInputs.js`